src/bonsai/calculators/accountActions.ts

```diff
@@ -88,7 +88,7 @@ export function createUsdcDepositOperations(
 ): SubaccountBatchedOperations {
   const updatedParentSubaccountData = addUsdcAssetPosition(parentSubaccount, {
     side: IndexerPositionSide.LONG,
-    size: depositAmount,
+    size: depositAmount ?? '0',
```

Comment:
The default here nullifies the thrown error below. I think we are safe to just remove it outright

Reply:
oh wait nvm nvm

Reply:
hmm should we keep it in case `depositAmount` is empty string? wondering why its fine to remove the error now that theres a default, when previously `depositAmount` was always defined 👀